### ISSUE TITLE
no audio in quickview player for mkv files

### ISSUE THREAD

**Issue by githubxnoob** [aleksey-hoffman/sigma-file-manager#155]
### Is there an existing issue for this?

- [X] I have searched the opened issues

### Problem description

![error image](https://i.imgur.com/rSLkXmI.png)

as you can see the audio button is not clickable. its only playing video and no audio

this only happens with .mkv files.

mp4 works

### Steps to reproduce

select any .mkv files and hit space. it will open quickview player and play the video without audio.

### Environment

App version: 1.4.0
App page: navigator
Operating System: x64, win32, 10.0.19042

**Comment by aleksey-hoffman** [comment]
I just tried opening an `mkv` video. Audio is working.
Try another video

**Comment by aleksey-hoffman** [comment]
I just tried opening an mkv video. Audio is working.
Try another video. Maybe this one has unsupported audio format

Also try opening it in a new tab of your Chrome and Edge browser (you can drag & drop the video there)

**Comment by Ansh4011** [comment]
@aleksey-hoffman Hi, I checked this out and it happened with one of my videos too. I suspect the number of channels to be the cause though, have tried with 2 `5.1` audio channel videos. The video which has audio in `5.1` channels wasn't showing any audio in SFM whereas the stereo audio was working perfectly fine.

**Comment by aleksey-hoffman** [comment]
@Ansh4011 thanks for testing. 
I have videos with different audio tracks

I could make it convert audio formats on the fly, but it would slow down preview loading and also create a duplicate file (or change the audio track in the original video)

**Comment by Ansh4011** [comment]
> I could make it convert audio formats on the fly, but it would slow down preview loading and also create a duplicate file (or change the audio track in the original video)

Indeed! Fast previews and ofcourse the consumption of less resources is important in today's time for any computer owner.

Btw, I guess there is some conflict between the audio codec and the preview script probably.

**Comment by aleksey-hoffman** [comment]
@Ansh4011 the app's UI is handled by Chromium. I have no control over it. If it cannot decode the audio, all I can do is convert the audio format of the video and then display that modified video instead.

**Comment by Ansh4011** [comment]
Oh, I get it. 
In that case I think it is better as it is right now. 
Thanks :)

**Comment by aleksey-hoffman** [comment]
Yeah, I'm not sure what else can be done.
@githubxnoob if you test the video in Google Chrome and it's not working there either, consider opening a bug report on Chromium website: https://bugs.chromium.org/p/chromium/issues/list

**Comment by RainyGrass** [comment]
Hello, the problem I am facing is exactly the opposite of yours. When my computer plays MKV files, the screen is blank and I can only hear the audio. My computer is Windows 11. How can I solve this problem?
![image](https://github.com/user-attachments/assets/03b2b249-0dcd-49cb-b748-70c6f6169993)
